### PR TITLE
added more restraints

### PR DESCRIPTION
**Pull request by wendywwang**
checked a few more discussions around reproducibility and added some more settings accordingly

also fixed the reflection pad2d that should have been changed

### CODE REVIEW

**Review by rich-nieto-ibm — APPROVED** [review]

File: deepliif/models/networks.py [hunk 1]
```diff
@@ -347,7 +347,7 @@ def __init__(self, input_nc, output_nc, ngf=64, norm_layer=nn.BatchNorm2d, use_d
         else:
             use_bias = norm_layer == nn.InstanceNorm2d
 
-        model = [nn.ReflectionPad2d(3), #nn.ReflectionPad2d(3),
+        model = [nn.ZeroPad2d(3) if os.environ['DEEPLIIF_SEED'] != 'None' else nn.ReflectionPad2d(3),
```
Comment (drewmibm) [inline deepliif/models/networks.py:350]: This was done to make the model deterministic?  Would this affect reloading saved models now that you are changing the layout of the network?

Comment (wendywwang) [reply deepliif/models/networks.py:350]: @drewmibm it will not, if you use the same setting when you run inference/reload (either put a seed or use no seed); but you are right, that means we need add this seed argument to the inference method as well

Comment (wendywwang) [reply deepliif/models/networks.py:350]: @drewmibm I think we can submit it for now, and see whether the decision is to stick to deterministic training (so these padding layers won't vary)

Comment (wendywwang) [reply deepliif/models/networks.py:350]: if not, we can then add this seed option to the inference method

**Comment by wendywwang** [comment]
@drewmibm merging it for now - if we need a change in inference, will create another PR; the current assumption is that deterministic training is an experimental feature, models trained using which are not supported yet in the inference method :p
